### DISCUSSION TITLE
Hiding obsolete documentation

### DISCUSSION THREAD

**Discussion by bullestock** [grblHAL/core#427] [General]
A Google search for "grblhal settings" returns https://github.com/terjeio/grblHAL/blob/master/doc/markdown/settings.md as the first hit, and that page is of course badly outdated. Would it be possible to hide or archive the old repo?

**Comment by terjeio** [comment]
I have now deleted the Wiki pages and cleared the doc files, and reachived the repo.

**Reply by bullestock** [reply]
Nice!
